contracts/SnappBase.sol

```diff
@@ -251,39 +252,32 @@ contract SnappBase is Ownable {
         
         claimableWithdraws[slot] = ClaimableWithdrawState({
             merkleRoot: _merkleRoot,
-            claimedBitmap: includedBitMap,
+            claimedBitmap: nullArray,
```

Comment:
Are you sure nullArray is copied here? If it was just passed by reference, we would share the same array with all different WithdrawStates.

I think it would be clearer to create a new array everytime

```suggestion
            claimedBitmap: new bool[](100),
```

This would also require to declare the type of `claimedBitmap:` to `bool[]` instead of `bool[100]`

Reply:
Is this a concern for storage purposes or is this actually better.  Also here we could use the constant from above MAX WITHDRAW SIZE

Reply:
This gives a compiler error...

```
TypeError: Invalid type for argument in function call. Invalid implicit conversion from bool[] memory to bool[100] memory requested.
            claimedBitmap: new bool[](100),
```

Reply:
This works, but I wasn't able to "cast" its size down

```
        bool[100] memory nullArray;

        claimableWithdraws[slot] = ClaimableWithdrawState({
            merkleRoot: _merkleRoot,
            claimedBitmap: nullArray,
            appliedAccountStateIndex: stateIndex()
        });
```